components/MessageOverlay.tsx

```diff
@@ -10,11 +10,7 @@ export const MessageOverlay: React.FC<MessageOverlayProps> = props => {
   const {t} = useTranslation('common');
   const style = StyleSheet.create({
     customHeight: {
```

Comment:
I still see occurance of customHeight in ScanLayoutController.ts(line 95) and App.tsx(Line 97). Please have a look.

Reply:
fixed and updated the pr